### ISSUE TITLE
convert keras to tensorflow

### ISSUE THREAD

**Issue by leeqiaogithub** [Microsoft/MMdnn#620]
when i use:mmconvert -sf keras xxx.hdf5 -df tensorflow -iw xxx.hdf5 -om xxx.pb
there is an error:
VauluError:Unknown loss function:earth_mover_loss.
this loss function was defined during training.
I am confused why you need a loss function in the transformation.

**Comment by rainLiuplus** [comment]
Hi @leeqiaogithub , mmdnn only support convert the network architecture and you can extract the network part by specifying the input node name and output node name using --inNodeName and --dstNodeName. In addition, would you mind past the error log here? Thanks!

**Comment by angyee** [comment]
> Hi @leeqiaogithub , mmdnn only support convert the network architecture and you can extract the network part by specifying the input node name and output node name using --inNodeName and --dstNodeName. In addition, would you mind past the error log here? Thanks!

@leeqiaogithub @rainLiuplus @kant  how to convert the .h5  to  .pb ?
